theseus/labs/lie/types.py

```diff
@@ -8,7 +8,9 @@
 class ltype(Enum):
     SE3 = 0
     SO3 = 1
+    tgt = 2
 
 
 SE3 = ltype.SE3
 SO3 = ltype.SO3
+tgt = ltype.tgt
```

Comment:
Different Lie groups should have different types of tangent vectors.

Reply:
Can you elaborate? Do they differ in anything other than their dimension?

Reply:
Is this resolved? It would be useful to get feedback during alpha release on the tangent stuff and it's usage.

Reply:
Converged that we will likely remove tangent tensor stuff. Discussing in example/alpha PR.